assets/check

```diff
@@ -12,23 +12,33 @@ payload=$TMPDIR/ibm-container-resource
 cat > "$payload" <&0
 initial_version=$(jq -r '.source.version // ""' < "$payload")
 environment=$(jq -r '.source.environment // ""' < "$payload")
+echo "[INFO] ${environment}"
 type=$(jq -r '.source.type // "default"' < "$payload")
-if [ -n "${environment}" ] || [[ "${environment}" == "ng" ]]
-then
-    environment=${environment}.
+if [ -n "${environment}" ]; then
+    if [[ "${environment}" == "prod" ]] || [[ "${environment}" == "test" ]] || [[ "${environment}" == "dev" ]]; then
```

Comment:
If it's `prod`, then the url should be `https://containers.cloud.ibm.com/v1/kube-versions`

Reply:
Thanks for your review! I have modified it. So pls help review again, thanks!